src/initialization/FGLinearization.cpp

```diff
@@ -69,47 +65,88 @@ FGLinearization::FGLinearization(FGFDMExec * fdm, int mode)
     // state feedback
     ss.y = ss.x;
 
-    std::vector< std::vector<double> > A,B,C,D;
-    std::vector<double> x0 = ss.x.get(), u0 = ss.u.get();
-    std::vector<double> y0 = x0; // state feedback
-    std::cout << ss << std::endl;
+    x0 = ss.x.get();
+    u0 = ss.u.get();
+    y0 = x0; // state feedback
 
     ss.linearize(x0,u0,y0,A,B,C,D);
 
+    x_names = ss.x.getName();
+    u_names = ss.u.getName();
+    y_names = ss.y.getName();
+    x_units = ss.x.getUnit();
+    u_units = ss.u.getUnit();
+    y_units = ss.y.getUnit();
+}
+
+void FGLinearization::GetStateSpace(Vector2D<double> & A_,
+                                    Vector2D<double> & B_,
+                                    Vector2D<double> & C_,
+                                    Vector2D<double> & D_) const {
+    A_ = A;
+    B_ = B;
+    C_ = C;
+    D_ = D;
+}
+
+std::vector<double> FGLinearization::GetInitialState() const {
+    return x0;
+};
+
+std::vector<double> FGLinearization::GetInitialInput() const {
+    return u0;
+};
+
+std::vector<double> FGLinearization::GetInitialOutput() const {
+    return y0;
+};
+
+std::vector<std::string> FGLinearization::GetStateNames() const {
+    return x_names;
+}
+
+std::vector<std::string> FGLinearization::GetInputNames() const {
+    return u_names;
+}
+
+std::vector<std::string> FGLinearization::GetOutputNames() const {
+    return y_names;
+}
+
+std::vector<std::string> FGLinearization::GetStateUnits() const {
+    return x_units;
+}
+
+std::vector<std::string> FGLinearization::GetInputUnits() const {
+    return u_units;
+}
+
+std::vector<std::string> FGLinearization::GetOutputUnits() const {
+    return y_units;
+}
```

Comment:
Such trivial methods should be inlined in the header. I suggest to remove all of them from `FGLinearization.cpp`
```suggestion
```